k8s/kube-base/xdmod_init.json

```diff
@@ -42,6 +42,11 @@
             "name": "nerc_openshift_staging",
             "formal_name": "NERC OpenShift Staging",
             "type": "hpc"
+        },
+        {
+            "name": "nerc_openshift_prod",
+            "formal_name": "NERC OpenShift Prod",
+            "type": "hpc"
         }
```

Comment:
This creates a resource in /etc/xdmod/resource.json.  It controls what the resource is called in xdmod.

when this changes, we need to manually get the changes through by running xdmod-init, or by directly running xdmod-setup.

I'm fine with this now, though I would prefer to have staging setup the same as production.

Reply:
Well, we can always redeploy with this change in staging after it's merged, correct?

Reply:
That's why I am fine with it.